code/processes/rdb.q

```diff
@@ -192,8 +192,6 @@ reload:.rdb.reload
 
 .lg.o[`init;"searching for servers"];
 .servers.startup[];
```

Comment:
should we also get rid of this `.servers.startup` given we have `.servers.startupdepcycles` below which should do the same thing I think?

Reply:
Yip good point - will do that.